### PR TITLE
Feature/mcfadden8/47 unaligned fits data

### PR DESCRIPTION
**Pull request by mcfadden8**
Added support for fits data files that are not page-aligned

### CODE REVIEW

File: tests/median_calculation/random_vector_median_calculation.cpp [hunk 1]
```diff
@@ -145,8 +155,8 @@ int main(int argc, char **argv) {
       double x_intercept = x_start_dist(rnd_engine);
       double y_intercept = y_start_dist(rnd_engine);
 
-      double x_slope = x_beta_dist(rnd_engine) * plus_or_minus(rnd_engine) * 25;
-      double y_slope = y_beta_dist(rnd_engine) * plus_or_minus(rnd_engine) * 25;
+      double x_slope = x_beta_dist(rnd_engine) * plus_or_minus(rnd_engine) * 2;
```
Comment (mayagokhale) [inline tests/median_calculation/random_vector_median_calculation.cpp]: Please put comment about what the constant multiplier means.

Comment (mcfadden8) [reply tests/median_calculation/random_vector_median_calculation.cpp]: @KeitaRckPn, would you add a comment to this and just commit/push to this branch?

Thanks,

**Comment by KeitaRckPn** [comment]
@mcfadden8 
Could you add the following comments to the part because I'm not sure how to modify code before a pull request made by another person is approved.

// Changed to the const value 2 from 25 so that vectors won't access out of range of the cube with a large number of frames
// This is a temporary measures

**Comment by mcfadden8** [comment]
@KeitaRckPn and @mayagokhale, I have added the comments that were provided by Keita (thanks).

@KeitaRckPn, in the future, you can update an existing pull request (PR) by:

1. Check out the branch of the PR (in this case it is ```git clone -b feature/mcfadden8/47-unaligned-fits-data```
2. Make you changes
3. Commit and push your changes

Your pushed changes will automatically be added to the existing PR.

**Pull request merged**